### PR TITLE
Per request

### PR DESCRIPTION
**Pull request by tehhowch**
Refs #2991 

 - Per #3415 the explosion weapon won't actually do anything yet.
 - Asset PR is opened to the high-dpi plugin for the @2x image https://github.com/endless-sky/endless-sky-high-dpi/pull/38

Blend and xcf are here: https://github.com/tehhowch/modding-assets/tree/master/modified%20osprey

Suggestions & feedback welcomed

### CODE REVIEW

**Comment by Pointedstick** [comment]
What a beast! Feel free to give Captain Nate the Friendly Author hails, too.

File: data/persons.txt [hunk 1]
```diff
@@ -405,3 +353,181 @@ person "Tranquility"
...
+	personality
+		mining harvests opportunistic frugal unconstrained
+	phrase
+		phrase
+			"friendly author"
+	phrase
+		word
+			"Does it sound like I'm ordering a pizza?"
+			"I've made a lot of special modifications myself."
+			"I take orders from just one person: me."
```
Comment (Bladewood) [inline data/persons.txt]: I feel like "myself" would work better here.

**Comment by tehhowch** [comment]
Can do.

It'd be nice to handle `hostile author` as well, but that requires a bit of rework to handle (since if the ship has hails defined, it doesn't reference its government to get any hails, which is how "normal' ships change their hails).

```
phrase "hostile author"
    word
        "I can't believe you've done this."
        "Changes requested."
        etc
```

File: data/persons.txt (hunk 1 above)
Comment (tehhowch) [reply data/persons.txt]: https://society6.com/product/han-solo-i-take-orders-from-just-one-person-me_print

Comment (Bladewood) [reply data/persons.txt]: And *of course* it's a reference.

Comment (Pointedstick) [reply data/persons.txt]: It's a Star Wars reference; you'll have to take it up with Han Solo. 🙂

**Pull request merged**